src/FSharpVSPowerTools.Logic/FsiReferenceCommand.fs

```diff
@@ -174,32 +174,25 @@ type FsiReferenceCommand(dte2: DTE2, mcs: OleMenuCommandService) =
         new ProjectProvider(project, getProjectProvider, (fun _ -> ()), id)
 
     let generateReferenceFiles (project: Project) =
-        maybe {
-            let! project = Option.ofNull project
-            let! currentConfiguration = 
-                try Some (project.ConfigurationManager.ActiveConfiguration.ConfigurationName.ToLower())
-                with e -> 
-                    Logging.logError "[FsiReference] Cannot get current configuration name: %O" e
-                    None
-            let loadRefsFileName = sprintf "load-references-%s.fsx" currentConfiguration
+        Option.ofNull project
+        |> Option.iter (fun project ->
+            let loadRefsFileName = "load-references.fsx"
```

Comment:
Yes, it's. I'm just tired to commit the generated files each time I switch configuration and build.